### PR TITLE
ci(travis): speed up travis build

### PR DESCRIPTION
**Pull request by wd15**
Address #1075, #1077

 - [x] speed up travis by grouping nix-shell
 - [x] use app to send comment, not comment.sh so that github token is in app (travis can't use token from PRs not from usnistgov org)
 - [ ] ensure that staticman PR comment is written correctly for app to extract data (this PR needs to be merged before testing this)
 - [x] build Surge straight away and send message
 - [x] don't run `make *` for surge, it doesn't change anything anymore, it's a check not a backend thing
 - [x] update pull request template
 - [x] add bot account for posting comments instead of personal account
 - [x] ensure allow_origins is reset correctly after testing

Surge is currently deploying in less than 3 minutes after Travis starts up.

These changes can be [viewed live][live-site].

**Remember to tear down the [live site][live-site] when merging
or closing this pull request.**

[live-site]: https://random-cat-1089.surge.sh

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/1089)

### CODE REVIEW

**Comment by wd15** [comment]
testing comments

**Comment by wd15** [comment]
testing comments

**Comment by wd15** [comment]
test comment 1

**Comment by wd15** [comment]
test comment 1

**Comment by wd15** [comment]
test comment 2

**Comment by wd15** [comment]
test comment 2

**Comment by wd15** [comment]
test comment 2

**Comment by wd15** [comment]
test comment 2

**Comment by wd15** [comment]
test comment 2

**Comment by wd15** [comment]
test comment 2

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload! Your upload appears to have passed the tests.

You can view your upload display at

 - {domain}/simulations/display/?sim={comment.sim_name}

and

 - {domain}/simulations/{comment.benchmark_id}

Please review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link]({domain}/simulations/upload_form/?sim={comment.sim_name}).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload! Your upload appears to have passed the tests.

You can view your upload display at

 - https://random-cat-$1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-$1089.surge.sh/simulations/3a

Please review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-$1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload! Your upload appears to have passed the tests.

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload! Your upload appears to have passed the tests.

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload! Your upload appears to have passed the tests.

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload! Your upload appears to have passed the tests.

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload! Your upload appears to have passed the tests.

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
The new [PFHub live website]({comment.surge_domain}) is ready for
review.

**Comment by wd15** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by wd15** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload!

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please check that the tests pass below and then review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload!

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please check that the tests pass below and then review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
@wd15, thanks for your PFHub upload!

You can view your upload display at

 - https://random-cat-1089.surge.sh/simulations/display/?sim=test

and

 - https://random-cat-1089.surge.sh/simulations/3a

Please check that the tests pass below and then review and confirm your approval to @wd15 by commenting in this pull request.

If you think there is a mistake in your upload data, then you can resubmit the upload [at this link](https://random-cat-1089.surge.sh/simulations/upload_form/?sim=test).

**Comment by wd15** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by wd15** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by wd15** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by wd15** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by wd15** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by pfhub** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by pfhub** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

**Comment by pfhub** [comment]
The new [PFHub live website](https://random-cat-1089.surge.sh) is ready for review.

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -6,18 +6,8 @@ A longer description and justification for the changes.
 
 Raise any areas of concern.
 
+A comment will appear below when the live site is built. This normally
+takes a few minutes.
```
Comment (tkphd) [inline .github/PULL_REQUEST_TEMPLATE]: 10 to 15

Comment (wd15) [reply .github/PULL_REQUEST_TEMPLATE]: No, it takes a few minutes for the comment to appear. The 10 to 15 minutes is for it to finish the tests. The uploaders don't care about that so much.

Comment (wd15) [reply .github/PULL_REQUEST_TEMPLATE]: the comment appears in about 3 minutes

**Pull request merged**